lib/experimental/class-wp-menu-conversion-gutenberg.php

```diff
@@ -0,0 +1,150 @@
+<?php
+/**
+ * WP_Menu_Conversion_Gutenberg class
+ *
+ * @package gutenberg
+ * @since 6.3.0
+ */
+
+/**
+ * Manages Fallback behavior for Navigation menus.
+ *
+ * @access public
+ */
+class WP_Menu_Conversion_Gutenberg {
```

Comment:
```suggestion
class WP_Nav_Menu_Conversion_Gutenberg {
```

Consistency with naming of Core Classic Menu functions?

Reply:
I think having Nav or even Navigation in the name is important to make it clear this is not a classic menus thing.

Reply:
How about WP_Navigation_Menu_Converter_Gutenberg

Reply:
I wonder if this class should also work the other way around so we'd have WP_Navigation_Menu_Converter_Gutenberg::toBlocks($classic) and WP_Navigation_Menu_Converter_Gutenberg::toClassic($blocks)

Reply:
Possibly. But not in this PR. If we build the class correctly then it would be easy to extend it but I don't think i'll do it unless I have a use case to drive it.

Reply:
✅ Nav Menu is consistent with functions relating to classic menus such as `nav_menu()`.